src/radical/saga/adaptors/slurm/slurm_job.py

```diff
@@ -663,6 +662,8 @@ def _job_run(self, job_obj):
 
             if  'stampede2' in self.rm.host.lower() or \
                 'longhorn'  in self.rm.host.lower() or \
+                'spock'     in self.rm.host.lower() or \
```

Comment:
should we move it into the section below with just `-N` option? (since examples with Spock are not with `-n`)

Reply:
Sure - that's done now.